src/api/index.md

```diff
@@ -2101,14 +2101,14 @@ type: api
 - **Limited to:** directives that expect `Array` values, e.g. `v-for`
 
 - **Arguments:**
-  - `{String} sortKey`
+  - `{String | Array<String> | Function} sortKeys`
   - `{String} [order] - default: 1`
 
 - **Usage:**
 
-  Return a sorted version of the source Array. The `sortKey` is the key to use for the sorting. The optional `order` argument specifies whether the result should be in ascending (`order >= 0`) or descending (`order < 0`) order.
+  Return a sorted version of the source Array. `sortKeys` is an Array with the keys to use for the sorting. You can pass a String if you only need to sort on one key. You can also pass a Function if you want to use your own sorting strategy instead. The optional `order` argument specifies whether the result should be in ascending (`order >= 0`) or descending (`order < 0`) order.
```

Comment:
Some suggestions:

- ... Array. **The** `sortKeys` is ...
- You can pass a String if you need to sort only one key.